src/main/java/com/thealgorithms/strings/ReverseWordInString.java

```diff
@@ -0,0 +1,47 @@
+package com.thealgorithms.strings;
+
+/**
+ Given an input string s, reverse the order of the words.
+ A word is defined as a sequence of non-space characters.
+ The words in s will be separated by at least one space.
+ Return a string of the words in reverse order concatenated by a single space.
+ Note that s may contain leading or trailing spaces or multiple spaces between two words.
+ The returned string should only have a single space separating the words.Do not include any extra spaces
+ */
+
+public class ReverseWordInString {
+
+    public static void main(String[] args) {
+
+        System.out.println(reverseWords("the sky is blue"));
+    }
```

Comment:
Please remove the `main` method and add some proper tests. Good examples to look at:
- [`Factorial`](https://github.com/TheAlgorithms/Java/blob/566c27a996f48a94916170c159ae735546b39c42/src/main/java/com/thealgorithms/maths/Factorial.java) and [`FactorialTest`](https://github.com/TheAlgorithms/Java/blob/566c27a996f48a94916170c159ae735546b39c42/src/test/java/com/thealgorithms/maths/FactorialTest.java),
- [`FindMax`](https://github.com/TheAlgorithms/Java/blob/566c27a996f48a94916170c159ae735546b39c42/src/main/java/com/thealgorithms/maths/FindMax.java) and [`FindMaxTest`](https://github.com/TheAlgorithms/Java/blob/566c27a996f48a94916170c159ae735546b39c42/src/test/java/com/thealgorithms/maths/FindMaxTest.java).

```suggestion
```

Reply:
Thanks done require changes please review